### PR TITLE
Drop Python 2 support

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by leplatrem** [comment]
I'm struggling with this: 

``` 
  File "/home/travis/build/Cornices/cornice/cornice/service.py", line 493, in wrapper

    validator(request, **args)

  File "/home/travis/build/Cornices/cornice/tests/validationapp.py", line 193, in rebinding_validator

    return colander_body_validator(request, **kwargs)

  File "/home/travis/build/Cornices/cornice/cornice/validators/_colander.py", line 63, in _validator

    class RequestSchema(colander.MappingSchema, metaclass=RequestSchemaMeta):  # noqa

TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases

```

@Natim any idea?

File: cornice/validators/_colander.py [hunk 1]
```diff
@@ -62,8 +60,7 @@ def __new__(cls, name, bases, class_attrs):
                 class_attrs[location] = _ensure_instantiated(schema)
                 return type(name, bases, class_attrs)
 
-        class RequestSchema(with_metaclass(
-                RequestSchemaMeta, colander.MappingSchema)):
+        class RequestSchema(colander.MappingSchema, metaclass=RequestSchemaMeta):  # noqa
```
Comment (Natim) [inline cornice/validators/_colander.py]:
```suggestion
        class RequestSchema(colander.RequestSchemaMeta):
```

File: cornice/validators/_colander.py [hunk 2]
```diff
@@ -62,8 +60,7 @@ def __new__(cls, name, bases, class_attrs):
...
             """A schema to validate the request's location attributes."""
             pass
```
Comment (Natim) [inline cornice/validators/_colander.py]:
```suggestion
        class RequestSchema(colander.MappingSchema):
            __metaclass__ = RequestSchemaMeta
            """A schema to validate the request's location attributes."""
            pass

```

Comment (leplatrem) [reply cornice/validators/_colander.py]: Python2 style?

Comment (Natim) [reply cornice/validators/_colander.py]: According to six doc it is python3 style :joy:

Comment (Natim) [reply cornice/validators/_colander.py]: Oh no you are right, I misread: https://six.readthedocs.io/#six.add_metaclass

**Pull request merged**